.github/workflows/test-code.yml

```diff
@@ -27,34 +27,48 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: 
+        os:
           - ubuntu-latest
           - windows-latest
         graphqlversion:
-          - 4.0.2
-          - 4.1.0
-          - 4.2.0
-          - 4.3.0
-          - 4.4.0
+          - 5.0.0
     steps:
       - name: Checkout source
-        uses: actions/checkout@v2
-      - name: Setup .NET Core 5.0 SDK
-        uses: actions/setup-dotnet@v1
+        uses: actions/checkout@v3
+      - name: Setup .NET Core SDKs
+        uses: actions/setup-dotnet@v2
         with:
-          dotnet-version: '5.0.x'
+          dotnet-version: |
+            5.0.x
+            6.0.102
```

Comment:
Personally I would only test with 6.0 for this project.  If anything else, 3.1 which is also a LTS release, but not 5.0.